app/src/main/assets/country_metadata/PW.yml

```diff
@@ -1,3 +1,4 @@
 # Do not edit. Source files are in /res/country_metadata
+measurementSystem: [imperial, metric]
```

Comment:
I added [Palau](https://en.wikipedia.org/wiki/Palau) because it is a part of [Micronesia](https://en.wikipedia.org/wiki/Micronesia) (FM) which uses the imperial system because it is/was a territory of the US